source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -154,7 +154,7 @@ The upgrade process begins by removing the cluster you are going to upgrade from
 
 ##  Environment Specific Steps
 Some environments do have some additional requirements / checks that need to be confirmed prior to upgrading any of its clusters.
-We upgrade PTL/PTLSbox and Preview/Dev in-place, meaning we aren't building another cluster or switching in the Application Gateway unless necessary. If one of these environments must instead be rebuilt and swapped over, please refer to the Rebuilding AKS Clusters guide.
+We upgrade PTL/PTLSbox and Preview/Dev/ITHC in-place, meaning we aren't building another cluster or switching in the Application Gateway unless necessary. If one of these environments must instead be rebuilt and swapped over, please refer to the Rebuilding AKS Clusters guide.
```

Comment:
Can you add in brackets next to ITHC that this is only because we have one cluster currently